dist/css/arizona-bootstrap.css

```diff
@@ -6204,7 +6224,7 @@ a.close.disabled {
   width: 2rem;
   height: 2rem;
   vertical-align: -0.125em;
-  border: 0.25em solid currentColor;
+  border: 0.25em solid currentcolor;
```

Comment:
Never heard of this but, apparently it is a thing https://css-tricks.com/currentcolor/

Reply:
Yes, it looks like `currentcolor` is correct: https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword